cmd/event-logger/event_sender.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/sqs"
+)
+
+type sqsClient struct {
+	svc      *sqs.Client
+	queueURL string
+}
+
+type CloudWatchEvent struct {
+	Version    string          `json:"version"`
+	ID         string          `json:"id"`
+	DetailType string          `json:"detail-type"`
+	Source     string          `json:"source"`
+	Account    string          `json:"account"`
+	Time       time.Time       `json:"time"`
+	Region     string          `json:"region"`
+	Resources  []string        `json:"resources"`
+	Detail     json.RawMessage `json:"detail"`
+}
+
+type ChangeConfirmedEvent struct {
+	UID       string `json:"uid"`
+	ActorType string `json:"actorType"`
+	ActorUID  string `json:"actorUID"`
+}
+
+func (c sqsClient) SendImmaterialChangeConfirmed(ctx context.Context, lpaUID, actorType, actorUID string) error {
+	detail, _ := json.Marshal(ChangeConfirmedEvent{
```

Comment:
instead of redefining the event shapes, could we either have `?detailType=whatever&detail={json}` or `?detailType=whatever&detail.uid=x&detail.actorType=y` as the interface (I was thinking you could assume anything that isn't `detailType` is nested in `detail` but `uid-requested` nests again)

Reply:
and another option is to make this a `POST` and send a JSON body